starters/expo-zustand-styled-components/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TouchableOpacityStyled, TextStyled } from './Button.styles';
+
+interface ButtonProps {
+  title: string;
+  onPress: () => void;
+}
+
+const Button = ({ title, onPress }: ButtonProps) => {
```

Comment:
Usually React native developers don't use the React native's button but create their button component with a Touchable area, because the native button component looks different on android and ios